DotNetRu.Android/Localize.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Globalization;
+using System.Threading;
+using Android.Content;
+using Xamarin.Forms;
+using XamarinEvolve.Clients.Portable.Helpers;
+using XamarinEvolve.Clients.Portable.Interfaces;
+
+[assembly: Dependency(typeof(DotNetRu.Droid.Localize))]
+
+namespace DotNetRu.Droid
+{
+    public class Localize : ILocalize
+    {
+        public void SetLocale(CultureInfo ci)
+        {
+            Thread.CurrentThread.CurrentCulture = ci;
+            Thread.CurrentThread.CurrentUICulture = ci;
+        }
+        public CultureInfo GetCurrentCultureInfo()
+        {
+            var netLanguage = "en";
+            var androidLocale = Java.Util.Locale.Default;
+            netLanguage = AndroidToDotnetLanguage(androidLocale.ToString().Replace("_", "-"));
+            // this gets called a lot - try/catch can be expensive so consider caching or something
+            System.Globalization.CultureInfo ci = null;
+            try
+            {
+                ci = new System.Globalization.CultureInfo(netLanguage);
+            }
+            catch (CultureNotFoundException e1)
+            {
+                // iOS locale not valid .NET culture (eg. "en-ES" : English in Spain)
+                // fallback to first characters, in this case "en"
+                try
+                {
+                    var fallback = ToDotnetFallbackLanguage(new PlatformCulture(netLanguage));
+                    ci = new System.Globalization.CultureInfo(fallback);
+                }
+                catch (CultureNotFoundException e2)
+                {
+                    // iOS language not valid .NET culture, falling back to English
+                    ci = new System.Globalization.CultureInfo("en");
+                }
+            }
+            return ci;
+        }
+        string AndroidToDotnetLanguage(string androidLanguage)
```

Comment:
We're not supporting all of this languages, why do we need this code?
Just fall back to "en", if current locale is not "ru"

Reply:
For example, if user has such languages, there might be exception.